pallets/identity/src/lib.rs

```diff
@@ -356,6 +356,41 @@ decl_module! {
             Ok(())
         }
 
+        // TODO: Remove this before mainnet
+        /// Register `target_account` with a new Identity and issue a CDD claim
+        ///
+        /// # Failure
+        /// - `origin` has to be a active CDD provider. Inactive CDD providers cannot add new
+        /// claims.
+        /// - `target_account` (primary key of the new Identity) can be linked to just one and only
+        /// one identity.
+        ///
+        /// # Weight
+        /// `7_000_000_000
+        #[weight = (7_000_000_000, DispatchClass::Normal, Pays::Yes)]
+        pub fn mock_cdd_register_did(
+            origin,
+            target_account: T::AccountId,
+        ) -> DispatchResult {
+            // Sender has to be part of CDDProviders
+            let cdd_id = Self::ensure_origin_call_permissions(origin)?.primary_did;
+            let cdd_providers = T::CddServiceProviders::get_members();
+            ensure!(cdd_providers.contains(&cdd_id), Error::<T>::UnAuthorizedCddProvider);
+
+            // Register a new identity for the target
+            let target_did = Self::_register_did(
+                target_account,
+                vec![],
+                Some(ProtocolOp::IdentityCddRegisterDid)
+            )?;
```

Comment:
Instead of this copy-paste, you could be calling `cdd_register_did` or some extracted version of it (which returns the DID).